R/transformCounts.R

```diff
@@ -24,12 +24,22 @@
 #' @param name \code{Character scalar}. A name for the column of the 
 #'   \code{colData} where results will be stored. (Default: \code{"method"})
 #'
-#' @param ... additional arguments passed on to \code{vegan:decostand}:
+#' @param ... additional arguments.
 #' \itemize{
-#'   \item \code{reference}: \code{Character scalar}. use to
+#'   \item \code{reference}: \code{Character scalar}. Used to
 #'   to fill reference sample's column in returned assay when calculating alr. 
 #'   (Default: \code{NA})
 #'   \item \code{ref_vals} Deprecated. Use \code{reference} instead.
+#'   \item \code{css.percentile}: \code{Numeric scalar}.  Used to set the 
+#'   percentile value that calculates the scaling factors in the css normalization.
+#'   (Default: \code{"0.5"})
```

Comment:
This should be numeric. "0.5" is character

Reply:
ACK